scripts/staking.py

```diff
@@ -90,14 +101,36 @@
                 print(f"Service {args.service_id} is not staked.")
                 sys.exit(0)
 
+            if is_service_evicted(
+                ledger_api, args.service_id, args.staking_contract_address
+            ):
+                print("WARNING: Your service has been evicted from the staking program due to inactivity.")
+                input("Press Enter to continue...")
+
             next_ts = get_next_checkpoint_ts(ledger_api, args.staking_contract_address)
+            ts_start = get_service_info(
+                ledger_api, args.service_id, args.staking_contract_address
+            )[3]
 
             liveness_period = get_liveness_period(
                 ledger_api, args.staking_contract_address
             )
             last_ts = next_ts - liveness_period
             now = time.time()
 
+            minimum_staking_duration = get_min_staking_duration(
+                ledger_api, args.staking_contract_address
+            )
+            available_rewards = get_available_rewards(
+                ledger_api, args.staking_contract_address
+            )
+
+            if (now - ts_start) < minimum_staking_duration and available_rewards > 0:
```

Comment:
this is cannot be done in everest

Reply:
See my comment above. Any release of this repository including this methods will be linked to the env. variables corresponding to Alpine.